satellite/satellitedb/overlaycache_test.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/internal/dbutil/pgutil"
+	"storj.io/storj/internal/dbutil/pgutil/pgtest"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
```

Comment:
```suggestion
	"storj.io/storj/satellite/satellitedb/dbx"
```

Reply:
The 28 other places where this package is imported, it has the `dbx` alias.

Reply:
so this actually is an alias because the package of `storj.io/storj/satellite/satellitedb/dbx` is `satellitedb`.

Reply:
I see.
I checked it but I looked to the wrong package, so I thought that the package was already named `dbx`